wit/blob-types.wit

```diff
@@ -0,0 +1,35 @@
+// name of a container, a collection of objects.
+// The container name may be any valid UTF-8 string.
+type container-name = string
+
+// name of an object within a container
+// The object name may be any valid UTF-8 string.
+type object-name = string
+
+type timestamp = u64
```

Comment:
Why wasn't time stamp used for created_at in container-metadata and object-metadata?

Reply:
That was a miss from the spec. Will raise an issue and a PR to fix this.

Reply:
Actually, the spec has been updated to use `timestamp` everywhere. I copied the older wits, will fix them